assets/sass/05-blocks/button/_editor.scss

```diff
@@ -7,21 +7,47 @@
 	line-height: var(--button--line-height);
 	background-color: var(--button--color-background);
 	border-radius: var(--button--border-radius);
+	border: var(--button--border-width) solid var(--button--color-background);
 	padding: var(--button--padding-vertical) var(--button--padding-horizontal);
 
-	&:hover,
-	&:focus,
-	&.has-focus {
+	&:visited {
```

Comment:
Did you try using `%extend button-style` here and migrating as much as possible to that mixin for ease of maintenance? I wonder why we didn't do that before 🤔